### PR TITLE
Merge kirkstone patches into master

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
This already passed [here](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/634237955) and I'm not planning to rerun it. The commit checks will fail due to different checking rules back in 2020.

File: tests/meta-mender-beaglebone-yocto-ci/conf/layer.conf [hunk 1]
```diff
@@ -9,6 +9,6 @@ BBFILE_COLLECTIONS += "meta-mender-beaglebone-yocto-ci"
 BBFILE_PATTERN_meta-mender-beaglebone-yocto-ci = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-mender-beaglebone-yocto-ci = "99"
 
-LAYERSERIES_COMPAT_meta-mender-beaglebone-yocto-ci = "dunfell"
+LAYERSERIES_COMPAT_meta-mender-beaglebone-yocto-ci = "dunfell kirkstone"
```
Comment (lluiscampos) [inline tests/meta-mender-beaglebone-yocto-ci/conf/layer.conf:12]: We should remove `dunfell` from here. (we are not really using this file, but we should keep these consistent).

Comment (lluiscampos) [reply tests/meta-mender-beaglebone-yocto-ci/conf/layer.conf:12]: Similarly for `meta-mender-raspberrypi3-ci`

File: tests/meta-mender-ci/conf/layer.conf [hunk 2]
```diff
@@ -9,16 +9,16 @@ BBFILE_COLLECTIONS += "meta-mender-ci"
 BBFILE_PATTERN_meta-mender-ci = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-mender-ci = "10"
 
-LAYERSERIES_COMPAT_meta-mender-ci = "dunfell"
+LAYERSERIES_COMPAT_meta-mender-ci = "dunfell kirkstone"
```
Comment (lluiscampos) [inline tests/meta-mender-ci/conf/layer.conf:12]: Remove `dunfell` from here.

**Review by lluiscampos — APPROVED** [review]
Good to go :rocket: (or, almost!)

I commented on few remaining references to `dunfell`. We can either fix in this PR or merge and launch a follow-up PR - it is up to you.

**Comment by kacf** [comment]
Thanks!

**Pull request merged**